test/gpu_tests.jl

```diff
@@ -30,8 +30,8 @@ CUDA.allowscalar(false)
 const ArrayType = CuArray
 
 # For debugging on the CPU
-#const backend = CPU()
-#const ArrayType = Array
+# const backend = CPU()
```

Comment:
should uncomment

Reply:
that's for debugging, I think it shows up as a change because of the space. I'll get rid of it tho